receiver/kafkareceiver/kafka_receiver.go

```diff
@@ -189,6 +189,9 @@ func (c *kafkaTracesConsumer) Shutdown(context.Context) error {
 		return nil
 	}
 	c.cancelConsumeLoop()
+	if c.consumerGroup == nil {
```

Comment:
The panic in the issue shows that it happened during `kafkaLogsConsumer.Shutdown`. Can you add this check for `kafkaLogsConsumer` and `kafkaMetricsConsumer`?

Have you been able to reproduce in testing to show that this resolves the panic? (I see the panic happens on this call in the logs consumer, but I just want to confirm)

Reply:
That's great, thanks for including!